README.md

```diff
@@ -140,40 +136,32 @@ end
 
 ## "Simple" *is* an Aesthetic
 
-Again, with software, there's no accounting for taste--but it *is* possible to
-dispassionately analyze complexity. Here is Straitjacket's *raison d'etre*.
-It is a belief and an aesthetic, and it's fine if you don't like it:
+Again, with software, it is only possible to dispassionately analyze complexity.
+Straitjacket espouses an aesthetic, and it's fine if you disagree, but:
 
 **Minimally complex software is better than any alternative.**
 
-To the author of Straitjacket, this is really important. Without some
-complexity, nothing gets done. All human undertakings require *some* complexity.
-But even one scintilla of *unnecessary* complexity is not welcome. That's the
-aspiration of Straitjacket. Pragmatic, no?
-
+Without *some* complexity, nothing meaningful gets done. All human undertakings
+require data, side effects, and decisions. But *unnecessary* complexity is bad.
```

Comment:
Yes, it is _unnecessary_ complexity that is bad and the root of the problems in terms of managing and maintaining software codebases. This needs to be articulated above where I have commented and the definition of complexity requires updating too.